velox/expression/LambdaExpr.h

```diff
@@ -54,13 +61,23 @@ class LambdaExpr : public SpecialForm {
       VectorPtr& result) override;
 
  private:
+  /// Used to initialize captureChannels_ and typeWithCapture_ on first use.
   void makeTypeWithCapture(EvalCtx& context);
 
   std::shared_ptr<const RowType> signature_;
-  std::vector<std::shared_ptr<FieldReference>> capture_;
+  /// The inner expression that will be applied to the elements of the input
+  /// array/map.
   ExprPtr body_;
-  // Filled on first use.
-  std::shared_ptr<const RowType> typeWithCapture_;
+  /// List of field references to columns in the input row vector.
+  std::vector<std::shared_ptr<FieldReference>> capture_;
```

Comment:
I remember that these may have nulls in front for the lambda arguments. Is this the case? If so, it would be nice to mention that.

Reply:
Thank you for checking. I may be confusing this with something else.